docs/Gemfile.lock

```diff
@@ -4,17 +4,17 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.6)
-    em-websocket (0.5.1)
+    concurrent-ruby (1.1.9)
```

Comment:
Is these updates are required in gem file?

Reply:
I'm not sure. I just ran the `bundle update` command and this is what I got. To be sure it works for you checkout the branch and run the docs. These updates were automatically applied